ZeeHomeScreen/Classes/Components/SectionComposite/SectionCompositeViewController.swift

```diff
@@ -341,7 +357,6 @@ import Zee5CoreSDK
                 if config.epgScreenID != nil {
                     topDistanceConstraint?.constant = -64
                     prepareToggle()
-                    prepareEPGView()
```

Comment:
Don't prepare EPG view while loading the screen, but rather when used clicks on EPG button